07LessonSeven/in-class-project/css/my-style.css

```diff
@@ -0,0 +1,57 @@
+body {
+  width: 850px;
+  margin:auto;
+}
+
+section {
+  background-image: url('../img/nasa-background.jpg');
+  height:450px;
+  width:850px;
```

Comment:
This is kind of nitpicky but try to be consistent about adding spaces after `:` in your CSS properties